### ISSUE TITLE
bd not changing directorys as expected on ubuntu 19.04

### ISSUE THREAD

**Issue by rs-development** [vigneshwaranr/bd#43]
bd folder name does only echo the target directory but does not actually switch to it.

I am using ubuntu 19.10 and fish 3.0.2, the same problem does exist in bash. Is there something to make it change the directory instead of just printing the target directory?

**Comment by TimothyGu** [comment]
You have to do the following steps documented in the README:

```sh
echo 'alias bd=". bd -si"' >> ~/.bashrc
source ~/.bashrc
```

**Comment by NajwaLaabid** [comment]
It's not clear that these commands need to be run for ubuntu as well, since they're under 'Other OS' in the README. Would be cool if this step is stated more clearly.

**Comment by rautamiekka** [comment]
Yep, those instructions're unclear: I just went and got the DEB off Launchpad and it indeed has literally no scripts in it besides the program itself.

**Comment by balazser** [comment]
@rs-development it is because you need to source it to make executed in the current shell. otherwise, it changes the directory still but in the subshell.
